Submitted By:            zippo zippo@oppiz.net
Date:                    2009-09-23
Initial Package Version: 3.1.1
Upstream Status:         Unknown
Origin:                  zippo@oppiz.net
Description: Fixes missing headers, GCC-4.3.2 and corrects errors for Glib 2.10
diff -Naur OOO310_m19/soltools/mkdepend/def.h OOO310_m19.new/soltools/mkdepend/def.h
--- OOO310_m19/soltools/mkdepend/def.h	2008-04-10 18:05:14.000000000 +0000
+++ OOO310_m19.new/soltools/mkdepend/def.h	2009-09-23 01:17:01.913883308 +0000
@@ -154,7 +154,7 @@
 
 char			*copy();
 char			*base_name();
-char			*getline();
+char			*get_line();
 char			*isdefined();
 struct filepointer	*getfile();
 struct inclist		*newinclude();
diff -Naur OOO310_m19/soltools/mkdepend/main.c OOO310_m19.new/soltools/mkdepend/main.c
--- OOO310_m19/soltools/mkdepend/main.c	2008-07-22 17:24:34.000000000 +0000
+++ OOO310_m19.new/soltools/mkdepend/main.c	2009-09-23 01:18:09.160646833 +0000
@@ -548,7 +548,7 @@
  * Get the next line.  We only return lines beginning with '#' since that
  * is all this program is ever interested in.
  */
-char *getline(filep)
+char *get_line(filep)
 	register struct filepointer	*filep;
 {
 	register char	*p,	/* walking pointer */
diff -Naur OOO310_m19/soltools/mkdepend/parse.c OOO310_m19.new/soltools/mkdepend/parse.c
--- OOO310_m19/soltools/mkdepend/parse.c	2008-04-10 18:06:39.000000000 +0000
+++ OOO310_m19.new/soltools/mkdepend/parse.c	2009-09-23 01:18:54.183802453 +0000
@@ -45,7 +45,7 @@
 	register int	type;
 	boolean recfailOK;
 
-	while ((line = getline(filep))) {
+	while ((line = get_line(filep))) {
 		switch(type = deftype(line, filep, file_red, file, TRUE, symbols)) {
 		case IF:
 		doif:
@@ -168,7 +168,7 @@
 	register char	*line;
 	register int	type;
 
-	while ((line = getline(filep))) {
+	while ((line = get_line(filep))) {
 		switch(type = deftype(line, filep, file_red, file, FALSE, symbols)) {
 		case IF:
 		case IFFALSE:
diff -Naur OOO310_m19/sw/source/core/doc/docbm.cxx OOO310_m19.new/sw/source/core/doc/docbm.cxx
--- OOO310_m19/sw/source/core/doc/docbm.cxx	2008-09-04 10:19:07.000000000 +0000
+++ OOO310_m19.new/sw/source/core/doc/docbm.cxx	2009-09-23 12:32:08.011104664 +0000
@@ -32,7 +32,7 @@
 // MARKER(update_precomp.py): autogen include statement, do not remove
 #include "precompiled_sw.hxx"
 
-
+#include <stdio.h>
 #include <tools/pstm.hxx>
 #include <svx/linkmgr.hxx>
 #include <fmtanchr.hxx>
diff -Naur OOO310_m19/sw/source/filter/ww8/WW8TableInfo.cxx OOO310_m19.new/sw/source/filter/ww8/WW8TableInfo.cxx
--- OOO310_m19/sw/source/filter/ww8/WW8TableInfo.cxx	2009-04-21 09:51:46.000000000 +0000
+++ OOO310_m19.new/sw/source/filter/ww8/WW8TableInfo.cxx	2009-09-23 12:51:12.882511596 +0000
@@ -31,6 +31,7 @@
 // MARKER(update_precomp.py): autogen include statement, do not remove
 #include "precompiled_sw.hxx"
 
+#include <stdio.h>
 #include <iostream> 
 #include "WW8TableInfo.hxx"
 #include "swtable.hxx"
